components/icons/images/tier2/merge.svg

```diff
@@ -0,0 +1,19 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+  <defs>
+    <clipPath id="a">
+      <path d="M0 0h24v24H0z"/>
+    </clipPath>
+  </defs>
+  <g clip-path="url(#a)" data-name="24x24-merge-icon">
+    <path fill="#fff" d="M0 0h24v24H0z"/>
+    <g data-name="Group 5080">
+      <path fill="none" stroke="#565a5c" stroke-linecap="round" stroke-width="2" d="M12.6 9h8.029a2.153 2.153 0 012.153 2.153v8.893a2.153 2.153 0 01-2.149 2.154h-10.48A2.153 2.153 0 018 20.045v-4.823" data-name="Path 3547"/>
+      <path fill="none" stroke="#565a5c" stroke-linecap="round" stroke-width="2" d="M4.351 15.197h-.122A2.22 2.22 0 012 13.044V4.151a2.153 2.153 0 012.152-2.154h11.48a2.153 2.153 0 012.153 2.155v.964" data-name="Path 3548"/>
```

Comment:
Similar here... our icons should be ferrite (`#494c4e`), so not sure why this new shade of grey is getting used.

Reply:
Sounds like there might be some feedback for Christopher here, @matthe97w?

Reply:
I've just finished it! It's an easy fix